data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,103 @@ mission "Arachi Cartoonist 2"
+			`	"Honored to have you fight alongside us, our company is," the armored Kimek begins. "Told about our objective, I assume you have been. Break into the fort, we must, before up the time is. Ten minutes, we have, once started, the battle has." She tells you to follow her to the 'war tent,' one that's slightly larger than the other ones around, and gives you a long piece of blue cloth. She then informs you that it is meant to be worn, so that others can tell what team you are on. Wrapping it around like a scarf, you look around the main tent to see that stored there is a stumpy log, reinforced with metal bits, which the Kimek tells you will be used as a battering ram to force open the gates. She then has you follow her to the edge of the camp, showing you the limits of this "set". It is marked by patches of vegetation, meant to indicate the boundaries of the game. "Also, discourage anyone from trying it, the thorns should, but against the rules, it is, to crawl up the walls of the fort, so-" She pauses, looking at you as if just realizing that such a thing is impossible for you. "Sorry, out of habit, I said it. Participate in this event, not many groundlocked people do, few Saryds here I've met. Even fewer humans, as imagine, you can."`
+			`	Once she's done, she says you're free to roam the camp as you wait for the "battle" to begin. A few minutes pass before a loud whistle blows, and the Kimek in charge of the camp tells everyone that it's begun. She leads the charge up the hill, with most of those from the camp following her. Others split into smaller groups and get closer to the trees that separate the sets, trying to approach the fort from the sides.`
+			choice
+				`	(Charge with the main group.)`
+					goto attw
+				`	(Take the longer side route.)`
+					goto attr
+			label attw
+			`	You follow the lead of the Kimek, and start a run for it up the hill, directly at the fort. Once you get closer, however, you see the defenders pouring down what looks like water. As it flows down the hill to meet your steps, you lose your balance and trip, sliding a bit down the hill. You get up, but try it as you may you cannot find footing with the grass, instead having to step on the few tufts that have grown too high for the liquid to cover them. With all their legs, the Kimek aren't so lucky, however, and continue slipping on the floor, with the Saryds in similar trouble. Only the Arachi can press onward, firmly holding onto the dirt beneath the grass with half their legs at a time. They slowly but surely help you and the others move up, and eventually you reach the gate, only to find that the groups that came from the sides are already working on busting it open. You can also see what the Kimek meant when she spoke of "thorns," with tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls.`
```

Comment:
```suggestion
			`	You follow the lead of the Kimek and start a run for it up the hill, directly at the fort. Once you get closer, however, you see the defenders pouring down what looks like water. As it flows down the hill to meet your steps, you lose your balance and trip, sliding a bit down the hill. You get up, but, try it as you may, you cannot find footing with the grass, instead having to step on the few tufts that have grown too high for the liquid to cover them. Even with all their legs, the Kimek aren't so lucky and continue slipping on the floor, with the Saryds in similar trouble. Only the Arachi can press onward, firmly holding onto the dirt beneath the grass with half their legs at a time. They slowly but surely help you and the others move up, and eventually you reach the gate, only to find that the groups that came from the sides are already working on busting it open. You can also see what the Kimek meant when she spoke of "thorns," with tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls.`
```
A comma fix, and changed the wording of the sentence describing how the Kimek and Saryds handle the slippery liquid for clarification.